plugins/tmux

```diff
@@ -0,0 +1 @@
+repository = https://github.com/aphecetche/asdf-tmux
```

Comment:
This URL should be the URL of the Git repository, which in this case would be `https://github.com/aphecetche/asdf-tmux.git`.

Reply:
Done.